modules/common/CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ TEMPLATE:
 
 ## [UPCOMING]
 ### Added
```

Comment:
same as in the previous changelog. Apply the same format to all the changelog files